src/backend/InvenTree/company/models.py

```diff
@@ -56,9 +54,21 @@ def rename_company_image(instance, filename):
     return os.path.join(base, fn)
 
 
+class CompanyReportContext(report.mixins.BaseReportContext):
+    """Report context for the Company model.
+
+    Attributes:
+        name: The name of the Company
+        primary_address: The primary address associated with the Company
+    """
+
+    name: str
+
+
 class Company(
     InvenTree.models.InvenTreeAttachmentMixin,
     InvenTree.models.InvenTreeNotesMixin,
+    report.mixins.InvenTreeReportMixin,
```

Comment:
When adding this mixin, a short code for a company report needs to be defined.

Reply:
Isn't the short code for the barcode mixin?